frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -108,47 +178,40 @@ const HostSummary = ({
   renderActionButtons,
   deviceUser,
   osSettings,
+  hostMdmDeviceStatus,
 }: IHostSummaryProps): JSX.Element => {
   const { status, platform } = titleData;
 
   const renderRefetch = () => {
```

Comment:
Not necessarily for this PR, but it seems like the entire `renderRefetch` could be moved out to the `<RefetchButton />` component.

Reply:
yep. I thought of that but didnt wanna add more work. Will do that in the future.